blocks/workfront-login/workfront-login.css

```diff
@@ -0,0 +1,138 @@
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.workfront-login {
+  --error: #c9252d;
+
+  width: 275px;
+  margin: auto;
+}
+
+.workfront-login .subdomain-wrapper {
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.workfront-login input {
+  background-color: #fff;
+  color: #4b4b4b;
+  box-sizing: border-box;
+  border: 1px solid;
+  border-radius: 4px;
+  text-indent: 0;
+  padding-left: 8px;
+  width: 100%;
+  height: 32px;
+  margin: 0;
+  overflow: visible;
+  font-size: 14px;
+  text-overflow: ellipsis;
+  transition: border-color .13s ease-in-out, box-shadow .13s ease-in-out;
+  outline: none;
+  appearance: none;
+}
+
+.workfront-login input:focus,
+.workfront-login input:hover,
+.workfront-login input:visited:hover {
+  border-color: var(--link-color-dark);
+  outline: none;
+}
+
+.workfront-login .subdomain-input {
+  width: 160px;
+}
+
+.workfront-login .subdomain-input.error {
+  border-color: var(--error);
+  padding-right: 35px;
+  background-repeat: no-repeat;
+  background-position: center right calc(35px / 4);
+  background-size: calc(35px / 2) calc(35px / 2);
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23eb1000' viewBox='-2 -2 7 7'%3e%3cpath stroke='%23d9534f' d='M0 0l3 3m0-3L0 3'/%3e%3ccircle r='.5'/%3e%3ccircle cx='3' r='.5'/%3e%3ccircle cy='3' r='.5'/%3e%3ccircle cx='3' cy='3' r='.5'/%3e%3c/svg%3E");
+}
+
+.workfront-login .input-group-append {
+  color: #505050;
+  font-size: 14px;
+  line-height: 1.5;
+  margin-left: 8px;
```

Comment:
Is there a need to reflect this for RTL? Same question for L31, L56, L84, L110

Reply:
Updated

RTL example: https://methomas-workfront-login--bacom--adobecom.hlx.live/ae_ar/drafts/methomas/workfront-login